keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/nonprobabilistic/operators/unary/ExpVertex.java

```diff
@@ -0,0 +1,26 @@
+package io.improbable.keanu.vertices.dbl.nonprobabilistic.operators.unary;
+
+import io.improbable.keanu.vertices.dbl.DoubleVertex;
+import io.improbable.keanu.vertices.dbl.nonprobabilistic.ConstantDoubleVertex;
+import io.improbable.keanu.vertices.dbl.nonprobabilistic.diff.DualNumber;
+import io.improbable.keanu.vertices.dbl.nonprobabilistic.diff.Infinitesimal;
+
+public class ExpVertex extends DoubleUnaryOpVertex {
```

Comment:
Can we add in a test for the dual number calculation? Either in the `DualNumbersTest` or make a `ExpVertexTest`